aspnetcore/tutorials/dotnet-watch.md

```diff
@@ -48,6 +48,8 @@ Navigate to the product API (`http://localhost:5000/api/math/product?a=4&b=5`),
  ```xml
  <ItemGroup>
  <DotNetCliToolReference Include="Microsoft.DotNet.Watcher.Tools" Version="1.0.0-msbuild3-final" />
+ <!-- If you use .NET Core 1.1.1 please check your version should be greater that 1.0.0-* -->
```

Comment:
Remove this line and leave the version to `1.0.0-*` that would be a sufficient change to have the correct information.